__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const app = require("../app");
+const pool = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const request = require("supertest");
+
+beforeEach(() => seed(testData));
+
+afterAll(() => {
+  pool.end();
+});
+
+describe("GET /api/topics endpoint", () => {
+  test("Responds with an array of topic objects with a slug and description property", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        topics = response.body;
+        //onsole.log(response.body)
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
+        expect(topics.length).toEqual(3);
+        expect(topics[0].slug).toEqual("mitch");
```

Comment:
be a little careful here, if we were to change the order of the topics coming back this would fail in the future, I think the arrays length coupled with the forEach is enough to give me confidence that this is working as intended